src/plugins/mf_priority.cpp

```diff
@@ -1078,8 +1078,8 @@ static int run_cb (flux_plugin_t *p,
 
 
 /*
- *  apply an update on a job with regard to its queue once it has been
- *  validated.
+ *  apply an update on a job with regard to its queue or associated bank once
```

Comment:
Commit message: " flux-accounting currently has no support for a user/bank to update the bank they want their pending job to run under." Should this just be "flux-accounting currently has no support for a user to update the bank of their job"?

Reply:
That's a better message than what I had suggested 😅 thanks! I've just updated this commit message.